src/serlio/prtMaterial/PRTMaterialNode.cpp

```diff
@@ -122,8 +118,8 @@ double MaterialInfo::getDouble(adsk::Data::Handle sHandle, const std::string& na
 	if (sHandle.setPositionByMemberName(name.c_str()))
 	{
 		double* data = sHandle.asDouble();
-		if (sHandle.dataLength() >= 1 && data != nullptr) {
-			return data[0];
+		if (sHandle.dataLength() >= 1 && data) {

```

Comment:
we kinda have the unwritten agreement in the team to do nullptr checks on raw pointers explicitely  (i.e. writing != nullptr)... for a smartptr that would have been correct.

Reply:
I see, I made all null pointer checks on raw pointers explicit throughout the whole project using `clang-tidy` with
```
clang-tidy -checks=-*,readability-implicit-bool-conversion -config="{Checks: '*', CheckOptions: [{ key: readability-implicit-bool-conversion.AllowIntegerConditions, value: 1}]} <source_files>"
```
We could think about adding `clang-tidy` to CI.

Reply:
yep, i'm a fan of clang-tidy as well... #38